CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ v0.22 + 1
   via `git_config_parse_path()` and `git_config_get_path()`
   respectively.
 
+* `git_index_add_frombuffer()` can now create a blob from memory
+   buffer and add it to the index, even if the index is also in
+   memory. The index must be attach to a repository to store the blob.
```

Comment:
This is contradictory. If the index is attached to a repository, it is not a purely in-memory index. I would just go with

>...add it to an index which is attached to a repository.